src/test/java/ua/lviv/iot/AppTest.java

```diff
@@ -0,0 +1,20 @@
+package ua.lviv.iot;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
```

Comment:
why this test is here? is it doing any work?

Reply:
Maven-archetype-quickstart created it by default. I will remove it.

Reply:
Fixed ✅